#! /bin/sh /usr/share/dpatch/dpatch-run
## 15_use_bash_in_ptp_camera_check.dpatch by  <fpeters@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Use bash since the script has bashisms

@DPATCH@
diff -urNad libgphoto2-2.3.1~/packaging/generic/check-ptp-camera libgphoto2-2.3.1/packaging/generic/check-ptp-camera
--- libgphoto2-2.3.1~/packaging/generic/check-ptp-camera	2006-12-24 16:16:19.000000000 +0100
+++ libgphoto2-2.3.1/packaging/generic/check-ptp-camera	2007-01-24 19:19:56.528663969 +0100
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
  
 # check if any interface of this device is a PTP camera interface (06)
 INTERFACE="${1:-06/01/01}"
